docs/api/python/function.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: docu
+title: Python Function API
+selected: Client APIs
+---
+
+You can create a DuckDB function out of a python function so it can be used in SQL queries.
+Just like regular [functions](../../sql/functions/overview) they need to have a name, a return type and parameter types.
+
+### Creating Functions
```

Comment:
Could we start off with an example at the top similar to what we do on other pages? Often that's what people are looking for in the docs.

Reply:
I've added another example using faker at the top :)